src/sources/aws_s3/mod.rs

```diff
@@ -34,7 +34,7 @@ pub enum Compression {
     /// The compression scheme of the object is determined from its `Content-Encoding` and
     /// `Content-Type` metadata, as well as the key suffix (for example, `.gz`).
     ///
-    /// It is set to 'none' if the compression scheme cannot be determined.
+    /// It is set to `none` if the compression scheme cannot be determined.
```

Comment:
```suggestion
    /// It is set to `None` if the compression scheme cannot be determined.
```

Reply:
Actually no - this isn't referring to a rust `None` it's referring to the `none` enum option.

Reply:
Reverted in [7fa8b61](https://github.com/vectordotdev/vector/pull/15836/commits/7fa8b61cfcbbb178bec95810849aed048ea47ba8)

Reply:
ah I missed the lowercase conversion 👍